Project.toml

```
@@ -0,0 +1,6 @@
+name = "Example"
+uuid = "7876af07-990d-54b4-ab0e-23690620f79a"
+version = "0.5.2"
+
+[compat]
+julia = "0.6, 0.7, 1"
```

Add `Test` dependency:
```suggestion
julia = "0.6, 0.7, 1"

[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"

[targets]
test = ["Test"]
```